robottelo/decorators.py

```diff
@@ -273,12 +273,18 @@ def bz_bug_is_open(bug_id):
     :rtype: bool
 
     """
+    upstream_mode = conf.properties.get('main.upstream', 1) == '1'
```

Comment:
Shouldn't this be:

```python
upstream_mode = conf.properties.get('main.upstream', '1') == '1'
```

That is, shouldn't we default to fetching `'1'` from the configuration file, not `1`?